lang/po/Firmware_sv.po

```diff
@@ -503,272 +503,272 @@ msgstr ""
 msgid "Info screen"
-msgstr ""
+msgstr "Infoskärm"
 
 # MSG_INSERT_FILAMENT c=20
 #: ultralcd.cpp:2482
 msgid "Insert filament"
-msgstr ""
+msgstr "Sätt i filament"
 
 # MSG_FILAMENT_LOADED c=20 r=2
 #: messages.c:38
 msgid "Is filament loaded?"
-msgstr ""
+msgstr "är filament isatt?"
 
 # MSG_STEEL_SHEET_CHECK c=20 r=2
 #: messages.c:109
 msgid "Is steel sheet on heatbed?"
-msgstr ""
+msgstr "Ligger metallskiva pa värmebädden?"
 
 # MSG_LAST_PRINT_FAILURES c=20
 #: messages.c:57
 msgid "Last print failures"
-msgstr ""
+msgstr "Senaste utskriftsfel"
 
 # MSG_WIZARD_WELCOME_SHIPPING c=20 r=16
 #: messages.c:122
 msgid "Hi, I am your Original Prusa i3 printer. I will guide you through a short setup process, in which the Z-axis will be calibrated. Then, you will be ready to print."
-msgstr ""
+msgstr "Hej, jag är din Prusa i3-skrivare. Jag guidar dig genom en kort installationsprocess, där Z-axeln kommer att kalibreras. Sedan är du redo att skriva ut."
 
 # MSG_ADDITIONAL_SHEETS c=20 r=9
 #: ultralcd.cpp:5029
 msgid "If you have additional steel sheets, calibrate their presets in Settings - HW Setup - Steel sheets."
-msgstr ""
+msgstr "Om du har ytterligare metallskivor, kalibrera deras förinställningar i Inställningar - HW inställning - Metallskiva."
 
 # MSG_LAST_PRINT c=18
 #: messages.c:56
 msgid "Last print"
-msgstr ""
+msgstr "Senaste utskrift"
 
 # MSG_SELFTEST_EXTRUDER_FAN c=20
 #: messages.c:89
 msgid "Left hotend fan?"
-msgstr ""
+msgstr "Vänst hotend fläkt?"
 
 # MSG_LEFT c=10
 #: ultralcd.cpp:2844
 msgid "Left"
-msgstr ""
+msgstr "Vänster"
 
 # MSG_BED_CORRECTION_LEFT c=14
 #: ultralcd.cpp:3114
 msgid "Left side [μm]"
-msgstr ""
+msgstr "Vänstsida [μm]"
 
 # MSG_LIN_CORRECTION c=18
 #: ultralcd.cpp:5702
 msgid "Lin. correction"
-msgstr ""
+msgstr "Linjär korrektion"
 
 # MSG_BABYSTEP_Z c=18
 #: messages.c:12
 msgid "Live adjust Z"
-msgstr ""
+msgstr "Live justera Z"
 
 # MSG_INSERT_FIL c=20 r=6
 #: ultralcd.cpp:7301
 msgid "Insert the filament (do not load it) into the extruder and then press the knob."
-msgstr ""
+msgstr "Infoga filamentet (ladda inte in det) i extrudern och tryck sedan pa knappen."
 
 # MSG_LOAD_FILAMENT c=17
 #: messages.c:58
 msgid "Load filament"
-msgstr ""
+msgstr "Ladda filament"
 
 # MSG_LOADING_COLOR c=20
 #: ultralcd.cpp:2516
 msgid "Loading color"
-msgstr ""
+msgstr "Laddar färg"
 
 # MSG_LOADING_FILAMENT c=20
 #: messages.c:59
 msgid "Loading filament"
-msgstr ""
+msgstr "Laddar filament"
 
 # MSG_ITERATION c=12
 #: messages.c:53
 msgid "Iteration"
-msgstr ""
+msgstr "Iteration"
 
 # MSG_LOOSE_PULLEY c=20
 #: ultralcd.cpp:8088
 msgid "Loose pulley"
-msgstr ""
+msgstr "Lossa pulley"
 
 # MSG_LOAD_TO_NOZZLE c=18
 #: ultralcd.cpp:6638
 msgid "Load to nozzle"
-msgstr ""
+msgstr "Ladd till munstyck"
 
 # MSG_M117_V2_CALIBRATION c=25
 #: messages.c:62
 msgid "M117 First layer cal."
-msgstr ""
+msgstr "M117 Först lager kalibr."
 
 # MSG_MAIN c=18
 #: messages.c:63
 msgid "Main"
-msgstr ""
+msgstr "Huvudsaklig"
```

Comment:
As in main menu, or just main. Huvudsaklig would be more like Mainly, no? So just "huvud". Additon: This is taking you back to the main menu, so Huvudmeny?